src/app/components/MainContent.tsx

```diff
@@ -31,22 +31,36 @@ const ContentStyles = styled(({ textSize, ...props }) => <DynamicContentStyles {
 
 // tslint:disable-next-line:variable-name
 const MainContent = React.forwardRef<HTMLDivElement, React.PropsWithChildren<Props>>(
-  ({book, children, className, ...props}, ref) => <Consumer>
-    {({registerMainContent}) => <main
-      ref={mergeRefs(ref, registerMainContent)}
-      className={className}
-      tabIndex={-1}
-    >
-      <ContentStyles
-        id={MAIN_CONTENT_ID}
-        book={book}
+  ({book, children, className, ...props}, ref) => {
+    const initialLoad = React.useRef(true);
+
+    React.useEffect(
+      () => {
+        if (initialLoad.current) {
+          initialLoad.current = false;
+        } else {
+          (ref as React.MutableRefObject<HTMLDivElement>).current?.focus();
```

Comment:
Just curious why the typecast is needed

Reply:
This ref comes in as a union of this type and a function that doesn't have `current`.
I'm open to the idea that I should [handle the function-ref case](https://stackoverflow.com/a/76739143), but this was my inclination.

Reply:
I have updated the code using the linked approach.